pkg/plugins/cherrypickapproved/cherrypickapproved.go

```diff
@@ -134,11 +134,19 @@ func (h *handler) handle(log *logrus.Entry, gc plugins.PluginGitHubClient, e git
 
 	// Filter configurations
 	foundRepoOrg := false
+	requireLGTMLabel := true
+	requireApprovedLabel := true
 	for _, cfg := range cfgs {
 		if cfg.Org == org && cfg.Repo == repo {
 			foundRepoOrg = true
 			approvers = cfg.Approvers
 			branchRe = cfg.BranchRe
+			if cfg.RequireLGTMLabel != nil && !*cfg.RequireLGTMLabel {
+				requireLGTMLabel = false
```

Comment:
No, because we want both variables to be true by default, and `*cfg.RequireLGTMLabel` would resolve to false if the option is not set. If we would do this, we would need defaulting in some other place (perhaps outside the plugin), which seems more complicated.

Reply:
Done. ✔️